tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -84,17 +84,71 @@ EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 # With 16 kHz DMIC1
 "sof-hda-generic\;sof-hda-generic-cavs25-4ch-48k-16k\;HDA_CONFIG=mix,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-cavs25-4ch-48k-16k.bin,\
-NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,DMIC1_RATE=16000"
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,\
+DMIC1_RATE=16000,DMIC1_PCM_ID=24"
 
 "sof-hda-generic\;sof-hda-generic-ace1-4ch-48k-16k\;HDA_CONFIG=mix,PLATFORM=mtl,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-ace1-4ch-48k-16k.bin,\
-NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,DMIC1_RATE=16000"
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,\
+DMIC1_RATE=16000,DMIC1_PCM_ID=24"
+
+# With 4ch 96 kHz DMIC1 and HDA
+"sof-hda-generic\;sof-hda-generic-cavs25-4ch-48k-96k\;HDA_CONFIG=mix,\
```

Comment:
it's the same here, now 96kHz applies to both HDAudio and DMIC...

And it'll be an absolute horror if we start adding heaphone specific streams, which can have a different rate to both the HDaudio speaker and DMIC paths...

Reply:
This is for development folder to show an example with multiple rates options, what the platform is capable to. The production topologies for ultrasound should have nicer simple names when the multi-rate architecture in decided.

Reply:
Development folder is fine - we have a chance to circle back for v2.10 and make improvements to overall topology suggested by @plbossart